internal/manifests/config/tempo-config.yaml

```diff
@@ -30,6 +51,13 @@ distributor:
             key_file: {{ .TLS.Paths.Key }}
             min_version: {{ .TLS.Profile.MinTLSVersionShort }}
 {{- end }}
+{{- if .ReceiverTLS.Enabled }}
```

Comment:
If `Gates.GRPCEncryption .Gateway` and `.ReceiverTLS.Enabled` is enabled, we end up with two `tls:` sections. Can you add a validation to the webhook to prevent enabling `ReceiverTLS` if `Gates.GRPCEncryption .Gateway` is enabled?